jobs/github-testers-macros.yaml

```diff
@@ -265,8 +265,8 @@
           TEST_DIR=`readlink -f test-results`
           chcon -Rt svirt_sandbox_file_t "$TEST_DIR"
           TESTS=(${{TESTS[@]}})
-          parallel -j100% sudo docker run --rm -v "$TEST_DIR":/junit "$CONTAINER" launch-test "{{}}" dnf ::: "${{TESTS[@]}}" || :
-          JSON=$(sudo docker run --rm "$CONTAINER" behave-2 --dry-run --no-summary -f json /behave/)
+          parallel -j100% sudo docker run --net=isolated --rm -v "$TEST_DIR":/junit "$CONTAINER" launch-test "{{}}" dnf ::: "${{TESTS[@]}}" || :
```

Comment:
@ignatenkobrain @mluscon @jsilhan @MichaelMraka  @mhatina I found the solution to disable networking and any inter-container communication for ours CI, but we have test that use 'dnf copr' command (test plugin-path-1) that requires internet connection, therefore I strongly recommend other disused solution (disable auto test for PRs from untrusted source) until we will be able to fix the test with copr.